tests/internal/src/combine_static_library_test/CMakeLists.txt

```diff
@@ -23,27 +35,28 @@ blt_combine_static_libraries( NAME FooStatic
                        LINK_POSTPEND ""
                      )
 
-blt_combine_static_libraries( NAME FooShared
-                       SOURCE_LIBS Foo1 Foo23
-                       LIB_TYPE SHARED
-                       LINK_PREPEND ""
-                       LINK_POSTPEND ""
-                     )
-
 blt_add_executable( NAME blt_combine_static_libraries_static_smoke
                     SOURCES blt_combine_static_libraries_static_smoke.cpp
                     DEPENDS_ON FooStatic gtest
                   )
-
 blt_add_test( NAME blt_combine_static_libraries_static_smoke
               COMMAND blt_combine_static_libraries_static_smoke
             )
 
-blt_add_executable( NAME blt_combine_static_libraries_shared_smoke
+if(NOT WIN32)
+    blt_combine_static_libraries( NAME FooShared
+                       SOURCE_LIBS Foo1 Foo23
+                       LIB_TYPE SHARED
+                       LINK_PREPEND ""
+                       LINK_POSTPEND ""
+                     )
+
+    blt_add_executable( NAME blt_combine_static_libraries_shared_smoke
                     SOURCES blt_combine_static_libraries_shared_smoke.cpp
                     DEPENDS_ON FooStatic gtest
```

Comment:
It seems like this should depend on ``FooShared``, right?

Reply:
The combine macro is unfortunately broken.  I have a new version from Burl that needs to be reviewed but I am pretty sure it works and solves the build dependency problem as well.